### ISSUE TITLE
Adding a back to top button

### ISSUE THREAD

**Issue by shrutiiigupta** [Vijaykv5/Image-Searcher#18]
**Description**
If we scroll down and then we want to go back to top, so rather than scrolling all the way up, a floating button to get to the top

**Expected Behavior**

Clicking on this button will scroll whole page to top

**Actual Behavior**

No such button exist

**Comment by Vijaykv5** [comment]
Yeah that's a great idea .... Thank you raising it @shrutiiigupta 

Are you interested in working on it

**Comment by shrutiiigupta** [comment]
Yes, I am interested

**Comment by jovyllebermudez** [comment]
This looks interesting. I can try this too.
